### DISCUSSION TITLE
How do I configure ESLint to support private properties class ?

### DISCUSSION THREAD

**Discussion by ericktucto** [eslint/eslint#17604] [Q&A]
I created my project with Vite and installed eslint with packages
- eslint
- eslint-config-prettier
- eslint-config-prettier
- eslint-config-vue
- prettier

My .eslintrc:
```json
{
  "root": true,
  "env": { "browser": true, "node": true },
  "extends": ["eslint:recommended", "plugin:vue/vue3-recommended", "prettier"],
  "plugins": ["prettier"],
  "rules": {
    "prettier/prettier": ["error"],
    "vue/require-default-prop": 0,
    "vue/html-indent": ["error", 4],
    "vue/singleline-html-element-content-newline": 0,
    "vue/component-name-in-template-casing": ["error", "PascalCase"]
  },
  "globals": { "_": true },
  "indent": ["error", 2]
}

```

Message error of ESLint:
![imagen](https://github.com/eslint/eslint/assets/22831024/f4b69360-8a82-43ca-b518-4632ae76c983)

**Comment by Rec0iL99** [comment]
Hi @ericktucto, include the `parserOptions` key with an `ecmaVersion` > `12` or `latest` for the latest version.

```js
parserOptions: {
    ecmaVersion: "latest",
  },
```

Plus, `indent` is a rule configuration and not a top level key. Move it into the `rules` key.

**Reply by ericktucto** [reply]
thanks your response, but ESLint does not yet recognize private property class
my .eslintrc
```json
{
  "root": true,
  "env": { "browser": true, "node": true },
  "extends": ["eslint:recommended", "plugin:vue/vue3-recommended", "prettier"],
  "plugins": ["prettier"],
  "parserOptions": {
    "ecmaVersion": "latest"
  },
  "rules": {
    "prettier/prettier": ["error"],
    "vue/require-default-prop": 0,
    "vue/html-indent": ["error", 4],
    "vue/singleline-html-element-content-newline": 0,
    "vue/component-name-in-template-casing": ["error", "PascalCase"],
    "indent": ["error", 2]
  },
  "globals": { "_": true }
}
```
![imagen](https://github.com/eslint/eslint/assets/22831024/d6cddf22-66b6-43e2-99d1-f463d16253d3)

**Reply by Rec0iL99** [reply]
Can you please provide a reproducible example for us to try out?

**Reply by ericktucto** [reply]
The error is not being thrown
![imagen](https://github.com/eslint/eslint/assets/22831024/d7f6e138-5388-46aa-806a-359b968f2891)
